internal/scanner/scanner.go

```diff
@@ -353,29 +358,30 @@ func (s *Scanner) auditResource(ctx context.Context, policies []*policies.Policy
 			if responseErr != nil {
 				errored = true
 				// log responseErr, will end in PolicyReportResult too
-				log.Error().Err(responseErr).Dict("response", zerolog.Dict().
-					Str("admissionRequest-name", admissionReviewRequest.Request.Name).
-					Str("policy", policy.GetName()).
-					Str("resource", resource.GetName()),
-				).Msg("error sending AdmissionReview to PolicyServer")
+				slog.Error("error sending AdmissionReview to PolicyServer",
+					slog.String("error", err.Error()),
```

Comment:
the original log was logging `responseErr` not `err.Error()`.
`err` in this case could be nil, and this is causing the test failure.